irmaconfig.go

```diff
@@ -59,6 +60,14 @@ type Configuration struct {
 	// (i.e., invalid signature, parsing error), and the problem that occurred when parsing them
 	DisabledSchemeManagers map[SchemeManagerIdentifier]*SchemeManagerError
 
+	// RequestorScheme data of the currently loaded requestorscheme
+	RequestorScheme *RequestorScheme
```

Comment:
This supports at most one requestor scheme simultaneously, unlike normal schemes, is that what we want?